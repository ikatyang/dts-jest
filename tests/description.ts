declare const description_1: () => {result: string};
declare const description_2: () => {result: number};
declare const description_3: () => {result: symbol};

// @dts-jest
description_1(

);

// @dts-jest
description_2(
);

// @dts-jest
description_3();
