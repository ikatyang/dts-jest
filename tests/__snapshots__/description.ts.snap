// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`description_1(
    
    ) 1`] = `"{ result: string; }"`;

exports[`description_2(
    ) 1`] = `"{ result: number; }"`;

exports[`description_3() 1`] = `"{ result: symbol; }"`;
